manage.py

```diff
@@ -0,0 +1,213 @@
+from flask_script import Manager
+from admin.dump_json import dump_lowlevel_json, dump_highlevel_json
+from admin.utils import create_path, remove_old_archives
+from datetime import datetime
+import acousticbrainz
+import subprocess
+import tempfile
+import psycopg2
+import tarfile
+import shutil
+import sys
+import os
+
+app = acousticbrainz.create_app()
+manager = Manager(app)
+
+db_connection = psycopg2.connect(app.config['PG_CONNECT'])
+
+# Importing of old dumps will fail if you change
+# definition of columns below.
+_tables = {
+    'lowlevel': (
+        'id',
+        'mbid',
+        'build_sha1',
+        'lossless',
+        'data',
+        'submitted',
+        'data_sha256',
+    ),
+    'highlevel': (
+        'id',
+        'mbid',
+        'build_sha1',
+        'data',
+        'submitted',
+    ),
+    'highlevel_json': (
+        'id',
+        'data',
+        'data_sha256',
+    ),
+    'statistics': (
+        'name',
+        'value',
+        'collected',
+    ),
+}
+
+
+@manager.command
+def init_db(archive=None):
+    """Initializes database and imports data if needed.
+
+    This process involves several steps:
+    1. Table structure is created.
+    2. Data is imported from the archive if it is specified.
+    3. Primary keys and foreign keys are created.
+    3. Indexes are created.
+
+    Data dump needs to be a .tar.xz archive produced by export command.
+
+    More information about populating a PostgreSQL database efficently can be
+    found at http://www.postgresql.org/docs/current/static/populate.html.
+    """
+    print('Creating tables...')
+    _run_sql_script('admin/sql/create_tables.sql')
+
+    if archive:
+        print('Importing data...')
+        _import_data(archive)
+
+    print('Creating primary and foreign keys...')
+    _run_sql_script('admin/sql/create_primary_keys.sql')
+    _run_sql_script('admin/sql/create_foreign_keys.sql')
+
+    print('Creating indexes...')
+    _run_sql_script('admin/sql/create_indexes.sql')
+
+    print("Done!")
+
+
+@manager.command
+def export(location=os.path.join(os.getcwd(), 'export'), threads=None, rotate=False):
+    print("Creating new archive...")
+    time_now = datetime.today()
+
+    # Creating a directory where dump will go
+    create_path(location)
+    archive_path = '%s/acousticbrainzdump-%s.tar.xz' % (location, time_now.strftime('%Y%m%d-%H%M%S'))
+    with open(archive_path, 'w') as archive:
+
+        pxz_command = ['pxz', '--compress']
+        if threads is not None:
+            pxz_command.append('-T %s' % threads)
+        pxz = subprocess.Popen(pxz_command, stdin=subprocess.PIPE, stdout=archive)
+
+        # Creating the archive
+        with tarfile.open(fileobj=pxz.stdin, mode='w|') as tar:
+            # TODO(roman): Get rid of temporary directories and write directly to tar file
+            temp_dir = tempfile.mkdtemp()
+
+            # Adding metadata
+            with open('%s/SCHEMA_SEQUENCE' % temp_dir, 'w') as f:
```

Comment:
Should I do this everywhere paths are constructed? For `archive_path` variable above and stuff below, for example.